src/gap.c

```diff
@@ -900,6 +900,40 @@ Obj FuncNanosecondsSinceEpoch(Obj self)
 
 /****************************************************************************
 **
+*F  FuncNanosecondsSinceEpochInfo( <self> )
+**
+**  'FuncNanosecondsSinceEpochInformation' returns a plain record
+**  contains information about the timers used for FuncNanosecondsSinceEpoch
+**
+*/
+Obj FuncNanosecondsSinceEpochInfo(Obj self)
+{
+  Obj res, tmp;
+  Int resolution;
```

Comment:
I'd hope that resolution does not fill a 32bit integer (its supposed to be the smallest possible increment in nanoseconds that one gets back between two calls to `NanosecondsSinceEpoch`), but at least in `timespec` it's potentially a `time_t` and a `long` (with maximum value of 999999999).

I also have to admit I never really looked at the typedefs for the different int types before, I'll change this to Int8.